container-scan/action.yaml

```diff
@@ -69,14 +59,123 @@ inputs:
   target:
     required: false
     description: Target build stage for multi-stage Docker builds
+  wiz-client-id:
+    required: true
+    description: Wiz Client ID for service account authentication
+  wiz-client-secret:
+    required: true
+    description: Wiz Client Secret for service account authentication
+  container-setup-commands:
+    required: false
+    description: Commands to set up the container environment
+  aws-access-key-id:
+    required: false
+    description: AWS Access Key ID
+  aws-secret-access-key:
+    required: false
+    description: AWS Secret Access Key
+
+outputs:
+  stdout:
+    description: The complete output from the Wiz CLI scan
+    value: ${{ steps.scan.outputs.stdout }}
+  has_findings:
+    description: Whether the scan found any issues
+    value: ${{ steps.scan.outputs.has_findings }}
+  scan_status:
+    description: Status of the scan (success/failure)
+    value: ${{ steps.scan.outputs.scan_status }}
+  vulnerabilities:
+    description: Summary of vulnerability counts
+    value: ${{ steps.scan.outputs.vulnerabilities }}
+  critical_count:
+    description: Count of critical vulnerabilities
+    value: ${{ steps.scan.outputs.critical_count }}
+  high_count:
+    description: Count of high vulnerabilities
+    value: ${{ steps.scan.outputs.high_count }}
+  medium_count:
+    description: Count of medium vulnerabilities
+    value: ${{ steps.scan.outputs.medium_count }}
+  low_count:
+    description: Count of low vulnerabilities
+    value: ${{ steps.scan.outputs.low_count }}
+  scan_url:
+    description: URL to the scan report
+    value: ${{ steps.scan.outputs.scan_url }}
+  image_id:
+    description: The Docker image ID (SHA256)
+    value: ${{ steps.docker-metadata.outputs.image_id }}
+  image_size:
+    description: The Docker image size in bytes
+    value: ${{ steps.docker-metadata.outputs.image_size }}
+  image_with_tag:
+    description: The full image name with tag
+    value: ${{ steps.docker-build.outputs.image-with-tag }}
 
 runs:
   using: composite
   steps:
     - name: Checkout Repository
-      if: inputs.checkout-repo == 'true'
       uses: actions/checkout@v4
 
+    - name: Download Wiz CLI
+      run: |
+        curl -o wizcli https://downloads.wiz.io/wizcli/latest/wizcli-linux-amd64
```

Comment:
As an improvement an to avoid hitting their servers every time this runs, we could add an optional S3 cache 

See how we use it here https://github.com/open-turo/action-pre-commit/blob/main/action.yaml#L97

This action is missing checking the cache hit, but we expose that as an output

Reply:
Can a specific version be specified when downloading it?